TMDBApp/TMDBApp/Genrer.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Genrer.swift
+//  TMDBApp
+//
+//  Created by Richard Leh on 15/01/2017.
+//  Copyright © 2017 Richard Leh. All rights reserved.
+//
+
+import Foundation
+
+enum ServerTMDBGenrer:String{
+    case id = "id"
+    case name = "name"
+}
+
+class Genrer{
```

Comment:
what I mean is: why is it named "Genrer"? In my opinion it seems to contain a Genre object. I'm not sure if Genrer would be some sort of a Genre generator, but in this case I'd expect to see a Movier class for consistency